### PR TITLE
[ch28811] Update libraries

### PR DESCRIPTION
**Pull request by emaciupe**
[ch28811]

### CODE REVIEW

File: src/etools_datamart/apps/etl/loader.py [hunk 1]
```diff
@@ -221,7 +221,7 @@ def contribute_to_class(self, model, name):
             self.config = model._etl_config
             del model._etl_config
             self.task = LoaderTask(self)
-            self.config.celery.tasks.register(self.task)
+            self.config.celery.register_task(self.task)
```
Comment (domdinicola) [inline src/etools_datamart/apps/etl/loader.py:224]: good catch!

Comment (domdinicola) [reply src/etools_datamart/apps/etl/loader.py:224]: https://github.com/celery/celery/discussions/7172

File: Pipfile [hunk 2]
```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 azure-storage-blob = "*"
-celery = "<5.2"  # error class based task
+celery = "==5.2.7"
```
Comment (domdinicola) [inline Pipfile:8]: update also `django-celery-beat`

Comment (emaciupe) [reply Pipfile]: done!

**Pull request merged**